app/views/govuk_publishing_components/components/_govspeak.html.erb

```diff
@@ -9,9 +9,24 @@
   classes << direction_class if direction_class
   classes << "disable-youtube" if disable_youtube_expansions
   classes << "gem-c-govspeak--inverse" if inverse
+
+  disable_ga4 ||= false
+
+  data_modules = "govspeak"
+  data_modules << " ga4-link-tracker" unless disable_ga4
+  data_attributes = { module: data_modules }
+
+  unless disable_ga4
+    data_attributes.merge!({
+      ga4_track_links_only: "",
+      ga4_limit_to_element_class: "call-to-action, info-notice, help-notice, advisory",
+      ga4_link: { "event_name": "navigation", "type": "callout" }.to_json,
+    })
+  end
+
 %>
 
-<div class="gem-c-govspeak govuk-govspeak <%= classes.join(" ") %>" data-module="govspeak">
+<%= tag.div(class: "gem-c-govspeak govuk-govspeak " + classes.join(" "), data: data_attributes) do %>
```

Comment:
Unrelated - I'm wondering if allowing classes prefixed with `direction-` should be allowed by the component wrapper.

Reply:
@andysellick I think as you created the component wrapper you have the final say 😄.  Though it's probably something that should be done in a separate PR - firstly because `direction_class` is passed through at the govspeak render time, so it might break things if somewhere in our code `direction_class` is passed through with a different prefix. Another issue is there's also the `disable-youtube` class which I guess would have to be refactored. So I guess it's out of the scope of this PR?

Reply:
Oh yes definitely - sorry, entirely non blocking comment.

Reply:
No worries!